src/reducers/books.js

```diff
@@ -0,0 +1,15 @@
+const books = (state = [], action) => {
```

Comment:
This is where we should use the dummy books that you were creating in `BookForm`